src/dune/merlin_server.ml

```diff
@@ -0,0 +1,128 @@
+open! Stdune
+open Import
+
+module Dot = struct
```

Comment:
It's easy to guess in this context, but I would still call this module `Dot_merlin`. Making the code obvious is generally better.